MAUI/Kanban-Board/Events.md

```diff
@@ -9,6 +9,122 @@ documentation: ug
 
 # Events in .NET MAUI Kanban Board (SfKanban)
 
+## CardTapped
+
+The `CardTapped` event occurs when the card is tapped. We can get the following details from the `CardTapped` event.
```

Comment:
@Kerubananthan3761 , add API links - https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Kanban.SfKanban.html#Syncfusion_Maui_Kanban_SfKanban_CardTapped

Reply:
Added API links in all places.